packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -155,10 +154,11 @@ describeNoCompat("GC data store sweep tests", (getTestObjectProvider) => {
 			summaryVersion,
 		);
 
-		const summarizerDataObject = await requestFluidObject<ITestDataObject>(
-			summarizingContainer2,
-			testDataObject.handle.absolutePath,
-		);
+		const containerRuntime = (summarizer2 as any).runtime as ContainerRuntime;
```

Comment:
FYI: I am not 100% sure if we should keep all usages of resolveHandle in GC tests. I created a task for the GC crew to look into this - AB#6332